src/python/WMCore/Services/Requests.py

```diff
@@ -301,7 +299,7 @@ def encode(self, data):
         """
         encode data into some appropriate format, for now make it a string...
         """
-        return urllib.urlencode(data, doseq=1)
+        return urllib.urlencode(data, doseq=True)
```

Comment:
If you look at urllib.urlencode documentation you'll see the following:
```
Help on function urlencode in module urllib:

urlencode(query, doseq=0)
    Encode a sequence of two-element tuples or dictionary into a URL query string.
```
So doseq is defined as integer, so don't change types when it is not required.

Reply:
It's actually a boolean in python3. In python2, it's not clear, they simply cite "if the optional parameter doseq is evaluates to True". So it's actually safer and forward compatible if we set it to a boolean.